src/Factory.php

```diff
@@ -11,6 +11,8 @@ class Factory
 
     public const DEFAULT_EXCLUDED_NAME = 'AcceptanceTesterActions.php'; // Annotated with @codingStandardsIgnoreFile
 
+    public const DEFAULT_INCLUDED_NAME = ['*.phtml', '*.phpt', '/\.php$/']; // PhpCsFixer adds *.php already (@todo: remove '/\.php$/' once https://github.com/symfony/symfony/issues/42675 fixed)
```

Comment:
Is .phpt no longer included by default?

Reply:
Correct, was removed a while ago, but seems useful to include.